gobblin-modules/gobblin-sql/src/main/java/gobblin/source/jdbc/SqlServerExtractor.java

```diff
@@ -187,7 +187,8 @@ public String getRightDelimitedIdentifier() {
   public String getConnectionUrl() {
     String host = this.workUnitState.getProp(ConfigurationKeys.SOURCE_CONN_HOST_NAME);
     String port = this.workUnitState.getProp(ConfigurationKeys.SOURCE_CONN_PORT);
-    String url = "jdbc:sqlserver://" + host.trim() + ":" + port;
+    String database = this.workUnitState.getProp(ConfigurationKeys.SOURCE_QUERYBASED_SCHEMA);
+    String url = "jdbc:sqlserver://" + host.trim() + ":" + port + ";databaseName=" + database.trim();
```

Comment:
Please check why existing code works for our SQLServer use cases and whether this change could break them.